main.nf

```diff
@@ -1575,6 +1591,7 @@ process multiqc {
     file (fastqc_trimmed:'fastqc/*') from ch_fastqc_results_trimmed.collect().ifEmpty([])
     file (host_removal) from ch_host_removed_log.collect().ifEmpty([])
     file ('quast*/*') from ch_quast_results.collect().ifEmpty([])
+    file ('bowtie2log/*') from ch_assembly_mapping_stats_for_multiqc.collect().ifEmpty([])
```

Comment:
Is this described within the MulitQC report, where these mappings actually come from? I didn't see any changes for this

Reply:
Yes, its in the same row. Perfectly aligned.